openjdk-24.yaml

```diff
@@ -227,7 +202,7 @@ subpackages:
     description: "Use the openjdk-24 JVM as the default JVM with the JDK installed"
     dependencies:
       runtime:
-        - ${{package.name}}-default-jvm
+        - java-common
         - ${{package.name}}
```

Comment:
We create the `default-jvm` in the openjdk-24-default-jvm package, do we want to create the same here, like we did in the openjdk-23 and earlier packages?

```yaml
    pipeline:
      - runs: |
          mkdir -p "${{targets.subpkgdir}}"/usr/lib/jvm
          ln -sf java-24-openjdk "${{targets.subpkgdir}}"/usr/lib/jvm/default-jvm
```

Reply:
Addressed in #39878